### ISSUE TITLE
output generator tweaks

### ISSUE THREAD

**Issue by missinglink** [pelias/api#152]
in the case of this document the generated `.text` property is `1 Main St, United Kingdom`, we should aim to have 3 components at all times where at least 2 'admin' properties are available.

this is simply to give the end-user enough information about similar results so that they can disambiguate and select the correct result.

expected: `1 Main St, Dungannon, United Kingdom`
actual: : `1 Main St, United Kingdom`

```javascript
  "features": [
    {
      "type": "Feature",
      "properties": {
        "id": "poi-address-osmnode-2656256259",
        "layer": "osmaddress",
        "name": "1 Main St",
        "alpha3": "GBR",
        "admin0": "United Kingdom",
        "admin1": "Dungannon",
        "category": [],
        "address": {
          "zip": "BT77 0BG",
          "number": "1",
          "street": "Main St"
        },
        "text": "1 Main St, United Kingdom"
      },
      "geometry": {
        "type": "Point",
        "coordinates": [
          -7.13167,
          54.428843
        ]
      }
    },
    {
```

**Comment by missinglink** [comment]
related? https://github.com/pelias/quattroshapes/issues/28

**Comment by orangejulius** [comment]
This will be fixed by pelias/fuzzy-tester#4 once merged :)

**Comment by missinglink** [comment]
@orangejulius this needs to be fixed in pelias/api as it's the code to produce the human readable text string
